jme3-terrain/src/main/resources/Common/MatDefs/Terrain/AdvancedPBRTerrain.j3md

```diff
@@ -279,6 +294,27 @@ MaterialDef Advanced PBR Terrain {
 
         Defines {
             BOUND_DRAW_BUFFER: BoundDrawBuffer
+
+            USE_FOG : UseFog
+            FOG_LINEAR : LinearFog
+            FOG_EXP : ExpFog
+            FOG_EXPSQ : ExpSqFog
+
+            EXPOSUREMAP : SunLightExposureMap
+            USE_VERTEX_COLORS_AS_SUN_EXPOSURE : UseVertexColorsAsSunIntensity
+            STATIC_SUN_EXPOSURE : StaticSunIntensity
+            BRIGHTEN_INDOOR_SHADOWS : BrightenIndoorShadows
+
+            USE_FIRST_LAYER_AS_TRANSPARENCY : FirstLayerAsTransparency;

```

Comment:
please remove the semicolon at the end of the line

Perhaps it would be better to rename the variable 'FirstLayerAsTransparency'  to 'UseFirstLayerAsTransparency' for consistency with the other variables (e.g. UseVertexColorsAsSunIntensity and useTriPlanarMapping)